### DISCUSSION TITLE
ToogaBooga v1.1.2

### DISCUSSION THREAD

**Discussion by ewang2002** [ToogaInc/ToogaBooga#157] [Announcements]
Hi everyone! We've officially released version 1.1.2 of ToogaBooga.  Here is a list of the relevant changes.

New commands introduced:
- listall: lists all users who have given role
- leaderboard: provides leaderboard of a given category
- leaderboardsync: one-time command to sync all users who have logged for a server, used for setting up the leaderboard (Moderator Only).

Command names changed:
- finduser -> find
- getstats -> stats
- addorchangename -> editname
- parsevc -> parse
- unsectionsuspend -> sectionunsuspend

Enhancements:
- logkey now separates exalt keys as a dropdown (including inc, vial, runes. etc)
- Elite loc introduced: can configure a section to output location to a text channel during an afk check
- AFK panels are left behind (like headcounts)
- Added Expose as a possible reaction

General Bugfixes
- ARL can use yoink and clean
- Securities can now use logparse
- Quota no longer splits between moderation roles (like helper and security)
- Helper perms added to raid instances
- Manual verification commands should no longer error due to unknown interaction, but may take a short while
- Verified role will no longer be removed immediately after being given
- Manual verification no longer gives double points

Thank you for your support as we continue to improve ToogaBooga. Our next goal is to polish the remaining rough edges and improve overall performance.
Please continue to use Tooga and provide feedback on anything you'd like to see added or improved.  Feel free to join our support Discord server, https://discord.gg/5fZu3asb4v, or use existing channels to communicate with us.

This release was brought to you by @dmansdman.

**Comment by dmansdman** [comment]
Note: Leaderboard commands are still a work in progress (very inefficient).  Recommend not using them for the time being.
